projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -2,19 +2,22 @@
   "name": "storefrontapp-e2e-cypress",
   "version": "1.0.0",
   "main": "index.js",
-  "author": "Marcin Lasak",
+  "author": "Spartacus team",
   "license": "MIT",
   "scripts": {
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:run:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/!(vendor)/**/*.e2e-spec.ts\"",
-    "cy:run:ci:all": "cypress run --config-file cypress.ci.json",
-    "cy:run:smoke": "cypress run --spec \"cypress/integration/smoke/**/*\"",
-    "cy:run:smoke:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/smoke/**/*\"",
+    "cy:run:ci:1905": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/!(vendor)/**/*.e2e-spec.ts\"",
+    "cy:run:ci:2005": "cypress run --config-file cypress.ci.2005.json --spec \"cypress/integration/!(vendor)/**/*.e2e-spec.ts\"",
+    "cy:run:ci:ccv2": "cypress run --config-file cypress.ci.ccv2.json --spec \"cypress/integration/!(vendor)/**/*.e2e-spec.ts\"",
     "cy:run:mobile": "cypress run --spec \"cypress/integration/mobile/**/*\"",
-    "cy:run:mobile:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/mobile/**/*\"",
+    "cy:run:mobile:ci": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/mobile/**/*\"",
```

Comment:
should we have a shortcut for the other environments as well like 2005 and ccv2 for mobile and regression to make it easier for all

Reply:
I am trying to only have/add the targets that we actually use for something. We can add them as we need them.